src/CiDetector.php

```diff
@@ -26,6 +26,7 @@ class CiDetector implements CiDetectorInterface
     public const CI_TEAMCITY = 'TeamCity';
     public const CI_TRAVIS = 'Travis CI';
     public const CI_WERCKER = 'Wercker';
+    public const CI_SOURCEHUT = 'SourceHut';
```

Comment:
Can you please sort it alphabetically? :)

Also below in `getCiServers()`

Reply:
Fixed in a rebased commit.